tests/post_training/test_quantizer_config.py

```diff
@@ -82,7 +82,7 @@ def test_default_quantizer_config(self, single_conv_nncf_graph):
         algo = PostTrainingQuantization()
         min_max_algo = algo.algorithms[0]
         min_max_algo._backend_entity = self.get_algo_backend()
-        q_setup = min_max_algo._get_quantizer_setup(single_conv_nncf_graph.nncf_graph, GraphPattern())
+        q_setup = min_max_algo._get_quantizer_setup(single_conv_nncf_graph.nncf_graph, GraphPattern(), GraphPattern())
```

Comment:
Pass `GraphPattern()` as keyword parameters explicitly, so that it is evident that there is no typos here.